compose.yaml

```diff
@@ -36,5 +36,28 @@ services:
       - ./db/pb_migrations:/pb_migrations
     restart: unless-stopped
 
+  meilisearch:
+    image: getmeili/meilisearch:latest
+    ports:
+      - "7700:7700"
+    volumes:
+      - ./volumes/meilisearch_data:/data.ms
+    restart: unless-stopped
+
+  search:
+    build:
+      context: ./search
+      dockerfile: Dockerfile
+    environment:
+      - PORT=4100
+      - PHX_HOST=localhost
+      - MIX_ENV=prod
+      - SECRET_KEY_BASE=1OhQxOL5gxafD09z/Eb0VNFw+qGInscrD2+oWgfXMsuYEafVFz/Jo+r+Vq4G3cOb
```

Comment:
No worries here with commiting the private key, when this gets deployed we just have to run `mix phx.gen.secret` to get a new one for the deployed environment